### PR TITLE
Upgrade dependencies to match Metadata v4 

### PR DESCRIPTION
**Pull request by Stefie**
For 1.0 release

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1762,7 +1762,7 @@
     "@types/memoizee" "^0.4.2"
     memoizee "^0.4.14"
 
-"@polkadot/api@^0.77.0-beta.4":
+"@polkadot/api@^0.77.0-beta.4", "@polkadot/api@^0.77.1":
```
Comment (amaurymartiny) [inline yarn.lock]: There shouldn't be any mention of `0.77.0-beta.4` inside yarn.lock anymore. Maybe update the resolutions field in package.json too?

Comment (Stefie) [reply yarn.lock]: Ah, dammit....

Comment (Stefie) [reply yarn.lock]: As in: Yes, totally. Should be fixed now

File: package.json [hunk 2]
```diff
@@ -10,9 +10,9 @@
     "packages/*"
   ],
   "resolutions": {
-    "@polkadot/api": "^0.77.0-beta.4",
+    "@polkadot/api": "^0.77.1",
     "@polkadot/keyring": "^0.76.1",
-    "@polkadot/types": "^0.77.0-beta.4",
+    "@polkadot/types": "^0.77.1",
```
Comment (Stefie) [inline package.json]: @amaurymartiny  It already is on 0.77.1 ?

File: package.json [hunk 3]
```diff
@@ -10,9 +10,9 @@
     "packages/*"
   ],
   "resolutions": {
-    "@polkadot/api": "^0.77.0-beta.4",
+    "@polkadot/api": "^0.77.1",
```
Comment (Stefie) [inline package.json]: api too

**Comment by amaurymartiny** [comment]
Ah sorry I wasn't clear, I was talking about https://github.com/polkadot-js/apps/blob/master/package.json#L2

**Pull request merged**